docs/content/misc/tasks.md

```diff
@@ -251,7 +251,6 @@ The indexing service can also run real-time tasks. These tasks effectively trans
       "maxRowsInMemory": 500000,
       "intermediatePersistPeriod": "PT10m",
       "windowPeriod": "PT10m",
-      "basePersistDirectory": "\/tmp\/realtime\/basePersist",
```

Comment:
@himanshug @rasahner this setting is ignored and Eric said it does that so that it can use the task dir for the task
so that it's storage is local to the task which is inline with what both of you said.